zip-frost.rst

```diff
@@ -0,0 +1,465 @@
+::
+
+  ZIP: Unassigned {numbers are assigned by ZIP editors}
+  Title: FROST for Spend Authorization Signatures
+  Owners: Conrado Gouvea <conrado@zfnd.org>
+          Chelsea Komlo <ckomlo@uwaterloo.ca>
+          Deirdre Connolly <deirdre@zfnd.org>
+  Credits: First Credited
+           ...
```

Comment:
If there are no credited people (excluding Owners), delete these lines. If there are, just give names, not email addresses.

Reply:
Done in https://github.com/zcash/zips/pull/662